src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/helper/WorkAllocationFieldsHelper.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.sscs.callback.handlers.helper;
+
+import static com.microsoft.applicationinsights.boot.dependencies.apachecommons.lang3.StringUtils.isNotBlank;
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.Benefit.getBenefitOptionalByCode;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.FormType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsType;
+
+public class WorkAllocationFieldsHelper {
```

Comment:
Not too sure these fields are for work allocation e.g. OGD type is SSCS5. I might be wrong but I think these are just the common components fields.

Reply:
The fields we are setting are in the WA fields class, the bug was raised from the WA tab in ExUI (I believe? Not sure.) So I assumed this was related to WA.

Reply:
Please confirm with the QA who raised it but I dont believe so. WA isnt a thing yet so I dont think they would have been testing it.